### ISSUE TITLE
ReliabilityProblem8 has a wrong LogNormal

### ISSUE THREAD

**Issue by mbaudin47** [mbaudin47/otbenchmark#16]
The current code is:
```
mu1 = 120.0
sigma1 = 12.0
X1 = ot.LogNormal(mu1, sigma1, 0.0)
```
This is wrong, because mu1 is the mean of the Log-Normal variable and sigma1 is the standard deviation of the Log-Normal variable.
The correct code is:
```
parameters = ot.LogNormalMuSigma(120.0, 12.0, 0.0)
X1 = ot.ParametrizedDistribution(parameters)
```

**Comment by mbaudin47** [comment]
Here are the faulty scripts:
* https://github.com/mbaudin47/otbenchmark/blob/master/examples/RP8.ipynb
* https://github.com/mbaudin47/otbenchmark/blob/master/otbenchmark/ReliabilityProblem8.py
* https://github.com/JEBROUN/otbenchmark/blob/FixStyle/examples/scripts/main19_07.py

**Comment by mbaudin47** [comment]
@JEBROUN : Please fix these parametrization issues

**Comment by mbaudin47** [comment]
@JEBROUN : the RP8.ipynb and ReliabilityProblem8.py are now ok. Could you please fix main19_07.py?

**Comment by JEBROUN** [comment]
it is done !!!

**Comment by mbaudin47** [comment]
In which PR?

**Comment by JEBROUN** [comment]
th PR name is : Exemple BBRC

**Comment by JEBROUN** [comment]
the PR name is : Example BBRC in the commit : " Parametrize LogNormal"

**Comment by JEBROUN** [comment]
I can't open the RP8.ipynb file in my directory !!!
![image](https://user-images.githubusercontent.com/43602056/82215941-6fab8d00-9918-11ea-88fb-e3ee4e6859a6.png)

**Comment by mbaudin47** [comment]
Please insert the issue number with the format "#XXX" in the corresponding PR, so that we can see the link between the issue number and the PR number.

**Comment by mbaudin47** [comment]
@JEBROUN : what PR number fixes the issue?

**Comment by mbaudin47** [comment]
Fixed at 29d351c6706c82dd4c8426efcfcccaef40c76114
